frontend/test/specs/inviteServiceSpec.js

```diff
@@ -35,7 +35,7 @@
             var result;
             service.sendInvite(inviteUser).then(function(data){
                 result = data;
-                body['data'] = {invite_body:inviteUser};
+                body['data'] = inviteUser;
```

Comment:
var promise = InviteService.sendInvite({invite_body: invite});

The inviteUser is not compatible with the real object received by service or is?

Reply:
Done.